### DISCUSSION TITLE
22-8-26的最新版本skynet编译lua部分报错

### DISCUSSION THREAD

**Discussion by jisuwudou** [cloudwu/skynet#1629] [Q&A] [answered]
make[2]: Leaving directory `/root/SkyNet/LearnSkynet/skynet-master/3rd/jemalloc'
cd 3rd/lua && make CC='cc -std=gnu99'
make[2]: Entering directory `/root/SkyNet/LearnSkynet/skynet-master/3rd/lua'
make[2]: *** No rule to make target `makefile', needed by `lapi.o'.  Stop.
make[2]: Leaving directory `/root/SkyNet/LearnSkynet/skynet-master/3rd/lua'
make[1]: *** [3rd/lua/liblua.a] Error 2
make[1]: Leaving directory `/root/SkyNet/LearnSkynet/skynet-master'
make: *** [linux] Error 2

我发现3rd/lua/makefile有更新就单独替换makefile 为老版本，然后能编译通过然后运行了，虽然能跑但实在是不知道哪里的问题。有老哥遇到这个问题么？

**Comment by cloudwu** [comment, marked as answer]
makefile 文件名大小写问题。你确定一下你目录下的 makefile 文件名是否是全小写。
